zipline/pipeline/graph.py

```diff
@@ -34,21 +33,23 @@ class TermGraph(DiGraph):
     Attributes
     ----------
     outputs
-    offset
-    extra_rows
 
     Methods
     -------
     ordered()
         Return a topologically-sorted iterator over the terms in self.
+
+    See Also
+    --------
+    ExecutionPlan
     """
     def __init__(self, terms):
-        super(TermGraph, self).__init__(self)
+        super(TermGraph, self).__init__()
```

Comment:
This was just a bug.  We were passing `self` twice to the parent constructor, which happened to work because the constructor of `DiGraph` takes another graph as its first argument, which is copied.  In this case we were forwarding an empty graph to the parent.